### DISCUSSION TITLE
Эффект подавления

### DISCUSSION THREAD

**Discussion by octothorpteam** [octothorp-team/public-dobrograd#540] [Предложения]
## ![](https://forum.octothorp.team//user_avatar/forum.octothorp.team/freedomchechnya/32/1265_2.png) [FreedomChechnya](https://forum.octothorp.team/u/FreedomChechnya) ([#5034](https://forum.octothorp.team/p/5034))

* * *

**DISCLAMER**

* * *

Всем привет! Эта тема уже была на старом форуме, я решил перенести ее сюда так как она затерялась и решения от администрации так и не последовало.

* * *

**В чем суть?**

* * *

Добавить некий эффект, который будет появляться тогда, когда в вашу сторону ведется
 огонь двумя или более человек, например потемнение в глазах или усиленную тряску рук, можно также громкий звук попадания самой пули, а лучше все сразу. Примечательно то, что эффект не будет действовать, или будет действовать не так сильно когда по вам стреляет малокалиберное оружие с небольшим темпом стрельбы.

* * *

**Для чего это вообще нужно?**

* * *

Часто замечал на сервере то, что игрокам просто плевать что по ним стреляют, они как Рэмбо стоят на открытом пространстве даже не думая подыскать укрытие. Эффект подавления позволит сковать цель в одном месте, не давая ей высунутся, что одинаково полезно как для полицейских, так и для преступников. Проще говоря этот эффект добавит реализма в перестрелки и заставит игрока думать, прежде чем он высунется из за укрытия перед превосходящими силами противника.

- За
- Против

0голосов

**Comment by octothorpteam** [comment]
## ![](https://forum.octothorp.team//user_avatar/forum.octothorp.team/4cb/32/176_2.png) [4CB](https://forum.octothorp.team/u/4CB) ([#5054](https://forum.octothorp.team/p/5054))


Как по мне это будет сложно реализовать, с нагрузкой на сервер, еще и пингом. Ну игрок сам в праве выбирать опускать оружие и сдаваться, продолжать стрельбу и умирать, или с шансом в 5% убить 3х полицейских в бронежилетах.

**Comment by octothorpteam** [comment]
## ![](https://forum.octothorp.team//user_avatar/forum.octothorp.team/пончик/32/1539_2.png) [ПоНчИк](https://forum.octothorp.team/u/ПоНчИк) ([#5055](https://forum.octothorp.team/p/5055))


добавь опрос в свой топик [@FreedomChechnya](/u/freedomchechnya)

**Comment by octothorpteam** [comment]
## ![](https://forum.octothorp.team//user_avatar/forum.octothorp.team/kyper/32/363_2.png) [Kyper](https://forum.octothorp.team/u/Kyper) ([#5197](https://forum.octothorp.team/p/5197))


ЗА, хьо нис луьш ву

**Comment by octothorpteam** [comment]
## ![](https://forum.octothorp.team//user_avatar/forum.octothorp.team/kolliderre/32/1363_2.png) [KolliDerre](https://forum.octothorp.team/u/KolliDerre) ([#5369](https://forum.octothorp.team/p/5369))


![](https://forum.octothorp.team/user_avatar/forum.octothorp.team/freedomchechnya/40/1265_2.png) FreedomChechnya:


> Добавить некий эффект, который будет появляться тогда, когда в вашу сторону ведется
>  огонь двумя или более человек, например потемнение в глазах или усиленную тряску рук, можно также громкий звук попадания самой пули, а лучше все сразу. Примечательно то, что эффект не будет действовать, или будет действовать не так сильно когда по вам стреляет малокалиберное оружие с небольшим темпом стрельбы.

По сути уже есть эффект при котором ты медленно идешь и у тебя идёт саунд как будто ты замерзаешь, или я не особо понял суть

**Comment by octothorpteam** [comment]
## ![](https://forum.octothorp.team//user_avatar/forum.octothorp.team/4cb/32/176_2.png) [4CB](https://forum.octothorp.team/u/4CB) ([#5453](https://forum.octothorp.team/p/5453))


Это при наведение пушки на человека. А он хочет когда в человека летят пули рядом(даже когда он за укрытием) и ему страшно от этого.

**Comment by octothorpteam** [comment]
## ![](https://forum.octothorp.team//user_avatar/forum.octothorp.team/satan/32/1296_2.png) [Satan](https://forum.octothorp.team/u/Satan) ([#5560](https://forum.octothorp.team/p/5560))


Я за.

**Comment by octothorpteam** [comment]
## ![](https://forum.octothorp.team//user_avatar/forum.octothorp.team/wine/32/1434_2.png) [Wine](https://forum.octothorp.team/u/Wine) ([#6002](https://forum.octothorp.team/p/6002))


странно

**Comment by octothorpteam** [comment]
## ![](https://forum.octothorp.team//user_avatar/forum.octothorp.team/zhilichni/32/641_2.png) [zhilichni](https://forum.octothorp.team/u/zhilichni) ([#7331](https://forum.octothorp.team/p/7331))


Интересная тема и я бы очень сильно хотел увидеть ее на сервере и посмотреть как ее реализуют.

**Comment by octothorpteam** [comment]
## ![](https://forum.octothorp.team//user_avatar/forum.octothorp.team/eagle/32/1834_2.png) [Eagle](https://forum.octothorp.team/u/Eagle) ([#8198](https://forum.octothorp.team/p/8198))


Я играя за копа постоянно стреляю на подавление дабы никто не высунулся из за укрытия, и это работает без всяких этих систем, так зачем она тогда нужна?

**Comment by Kradixsit** [comment]
Идея, конечно, интересная, но я бы предложил сделать так, чтобы у полицейских данного эффекта либо не было, либо он был бы не такой сильный.
